client/src/index.jsx

```diff
@@ -227,29 +242,26 @@ class App extends React.Component {
     }
 
     return (
-
-      <MuiThemeProvider muiTheme={muiTheme}>
-        <div>
-          <BrowserRouter>
-            <Switch>
-              <Route 
-                exact path="/signup" 
-                render={routeProps => <SignUp {...routeProps} logUserIn={this.logUserIn.bind(this)} />} 
-                />
-              <Route 
-                exact path="/login" 
-                render={routeProps => <Login {...routeProps} logUserIn={this.logUserIn.bind(this)} />} 
-                />
-              <Route 
-                path="/view?=(:id)" 
-                render={HomeWithProps} 
-                onEnter={ this.requireAuth }/>
-              <Route 
-                path="/" 
-                render={HomeWithProps} />
-            </Switch>
-          </BrowserRouter>
-        </div>
+      <MuiThemeProvider>
```

Comment:
Please make this line <MuiThemeProvider muiTheme={muiTheme}> to keep the color theme.